.github/workflows/minimal-console.yml

```diff
@@ -1,68 +1,92 @@
 name: Minimal console app build
 
 on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch name'
+        type: choice
+        options:
+          - harmstone
+          - master
+          - aarch64-pe-dev
+          - aarch64-pe-zac
   pull_request:
     paths-ignore:
       - '**/*.md'
   push:
     paths-ignore:
       - '**/*.md'
 
+env:
+  default_branch: harmstone  
```

Comment:
aarch64-pe-dev is more preferable branch to be default, otherwise it might be specified